account/views.py

```diff
@@ -1,3 +1,41 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import jwt
 
-# Create your views here.
+from django.views import View
+from django.http  import JsonResponse
+
+from .models     import User
+from my_settings import SECRET_KEY, ALGORITHM
+
+class UserSignUp(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            login_id     = data['login_id']
+            password     = data['password']
+            name         = data['name']
+            phone_number = data['phone_number']
+            email        = data['email']
+
+            if not login_id.isalnum():
+                return JsonResponse({"message":"No TUCKSOOMUNJA"}, status = 400)
+
+            if len(password) < 8:
```

Comment:
```suggestion
            if len(password) < MIN_PASSWORD_LENGTH:
```
8과 같은 비교값은 하드코딩 하지말고 상수로 지정해서 임포트 최 하단에 작성해주세요.

Reply:
비교값은 상수로 지정해서 임포트 최하단.. 메모.. 하겠습니다!!